### PR TITLE
Epic/manifest v3

### PR DESCRIPTION
**Pull request by Hatscat**
## 📝 What is this PR

**Describe what your PR addresses, could be a bug or a feature request**

-   [Specs](https://phantombuster.atlassian.net/browse/[TICKET-ID])

## 🏭 How does it work

**Briefly explain how you tackled the issue, what were the alternatives**

## 🗒 Notes

**Something you need to inform your reviewers about**

-   [ ] This change requires a documentation update
-   [ ] This change requires a unit tests update

## 🧪 QA Process

**Details a QA process to the reviewers**

### ✅ Positive testing (should succeed)

-   ...

### ❌ Negative testing (should fail)

-   ...

## 👌 To check

-   [ ] There's no regression
-   [ ] Coding guidelines are followed ([Contributing](https://github.com/phantombuster/engineering-docs/blob/master/contributing.md), [Testing](https://github.com/phantombuster/engineering-docs/blob/master/testing.md))

## 🚀 What to do after deployment

**Who should be notified after the deployment of this PR? What should be written in the changelog on Slack?**

### CODE REVIEW

**Comment by GabrielCousin** [comment]
@Hatscat double-checking two things:
- the goal is to merge into `master`, not `beta`, right?
- after the merge, are we going to release a new 1.7.1.x beta version or 1.7.0 stable?

**Comment by Hatscat** [comment]
> @Hatscat double-checking two things:
> 
> * the goal is to merge into `master`, not `beta`, right?
> * after the merge, are we going to release a new 1.7.1.x beta version or 1.7.0 stable?

* both of them, beta is for FF
* why "beta" release? "1.1.7.3" beta + prod I think

File: src/shared/sentry.ts [hunk 1]
```diff
@@ -1,15 +1,17 @@
 import * as Sentry from "@sentry/browser"
-import { browser } from "webextension-polyfill-ts"
+import * as browser from "webextension-polyfill"
 import { version } from "../../manifest.json"
```
[the remark this answers is not in the document]
Comment (GabrielCousin) [reply src/shared/sentry.ts:3]: well, I couldn't find where to skip calling `initSentry` and `Sentry.init` for development or beta releases (that's why I'm saying this)

File: manifest.json [hunk 2]
```diff
@@ -10,9 +10,9 @@
 	},
 	"homepage_url": "http://www.phantombuster.com/",
 	"background": {
-		"scripts": ["background.js"]
+		"service_worker": "background.js"
```
Comment (GabrielCousin) [inline manifest.json:13]: Tried in Firefox and it throws the following:

```
[ACTION FAILED] TEMPORARY_EXTENSION_INSTALL_FAILURE: background.service_worker is currently disabled
```

https://extensionworkshop.com/documentation/develop/manifest-v3-migration-guide/
Apparently, using `sw` is something specific to Chrome

Comment (GabrielCousin) [reply manifest.json:13]: l will make some tests tomorrow but it's likely we will be forced to maintain two manifest versions
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Background_scripts#implementing_background_scripts

Chrome:
```
	"background": {
		"service_worker": "background.js"
	}
```

Firefox:
```
"background": {
    "scripts": ["background-script.js"],
    "persistent": false
}
```

Comment (LoganPB) [reply manifest.json:13]: @LoganPB I was aware that there were differences between Chrome and Firefox for the manifest v3 but not which ones in detail. 
In this (post)[https://blog.mozilla.org/addons/2022/10/31/begin-your-mv3-migration-by-implementing-new-features-today/], They are saying Service Workers are not available "yet." 
So maybe later, we could have a manifest file for both. But when I was on it, the solution was to have 2 manifests and build the extension on the fly with commands for chrome or firefox.

File: beta-updates.json [hunk 3]
```diff
@@ -25,6 +25,10 @@
 				{
 					"version": "1.1.6.9",
 					"update_link": "https://github.com/phantombuster/web-browser-extension/releases/download/v1.1.6/firefox-1.1.6.9.xpi"
+				},
+				{
+					"version": "1.1.7.0",
```
Comment (GabrielCousin) [inline beta-updates.json:30]: could we confirm we don't need 1.1.7.3 here?

**Review by GabrielCousin — APPROVED** [review]

**Pull request merged**